cms/static/js/certificates/factories/certificates_page_factory.js

```diff
@@ -13,7 +13,7 @@ define([
         console.log('certificate_factory.function.start');
 
         var certificatesCollection = new CertificatesCollection(certificatesJson, {
-            parse: true, canBeEmpty: true
+            parse: true, canBeEmpty: true, certificateUrl: certificateUrl
```

Comment:
Unless the convention is to keep the attributes on a single line, I think it would be better to put each on its own line.